### ISSUE TITLE
Screen reader should read symbol inserted and cursor location otherwise

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#540]
### What are you trying to do that you can't?

When typing in the code editor, the screen reader does not communicate what symbol is being inserted, it only reads the cursor location.

### What is your idea?

The experience should generally mirror what happens in text fields:

* When inserting or deleting a symbol, the symbol should be read
* If there was a navigation, then the cursor location should be read

**Comment by amyjko** [comment]
Assigned!
